<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Chunk cx:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="cxSpinBox">
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="minimum">
        <number>-9000000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>cz:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="czSpinBox">
       <property name="minimum">
        <number>-9000000</number>
       </property>
       <property name="maximum">
        <number>9000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Chunk</string>
      </attribute>
     </widget>
     <widget class="NBTTreeView" name="nbtTreeView">
      <attribute name="title">
       <string>NBT Data</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NBTTreeView</class>
   <extends>QWidget</extends>
   <header>nbttreeview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
